x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/indices/flyout/steps/details/reindex_details_step.tsx

```diff
@@ -40,6 +40,30 @@ import { IndexClosedParagraph } from '../index_closed_paragraph';
 
 const ML_ANOMALIES_PREFIX = '.ml-anomalies-';
 
+const FollowerIndexCallout = () => (
+  <>
+    <EuiCallOut
+      title={
+        <FormattedMessage
+          id="xpack.upgradeAssistant.esDeprecations.indices.indexFlyout.detailsStep.followerIndexTitle"
+          defaultMessage="Unfollow leader index is recomended"
```

Comment:
This seems to align with the following docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-terminate-replication.html and https://staging-website.elastic.co/docs/deploy-manage/tools/cross-cluster-replication/ccr-terminate-replication, so I'd recommend similar terminology here:
```suggestion
          defaultMessage="Termination of replication is recommended"
```